src/main/java/uk/gov/hmcts/sptribs/common/event/page/PartiesToContact.java

```diff
@@ -50,11 +54,17 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseDa
         final CicCase cicCase = data.getCicCase();
         final List<String> errors = new ArrayList<>();
 
-        if (null != data.getContactParties() && CollectionUtils.isEmpty(cicCase.getNotifyPartySubject())
-            && CollectionUtils.isEmpty(cicCase.getNotifyPartyRepresentative())
-            && CollectionUtils.isEmpty(cicCase.getNotifyPartyApplicant())
-            && CollectionUtils.isEmpty(cicCase.getNotifyPartyRespondent())) {
-            errors.add("Which parties do you want to contact is required.");
+        if (cicCase != null) {
+            if (CollectionUtils.isEmpty(cicCase.getNotifyPartySubject())
```

Comment:
Can we import isEmpty as static import so we don't have to have CollectionUtils. before each?